apps/sertra/components/layout.tsx

```diff
@@ -6,7 +6,9 @@ export const Layout: FC<PropsWithChildren> = (props) => {
   return (
     <>
       <Navbar />
-      <div style={{ width: "1140px", marginLeft: "auto", marginRight: "auto" }}>
+      <div
+        style={{ maxWidth: "1140px", marginLeft: "auto", marginRight: "auto" }}
```

Comment:
Agreed, but I interpreted that we already have a separate ticket for this, so I intentionally touched it as little as I needed to make sure the changed done to listing view works as intended. I'm not sure if I should even change this in this PR at all, now that I think of it, as it might have adverse effects on other views.